src/test/java/LoginTests.java

```diff
@@ -0,0 +1,83 @@
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LoginTests extends BaseTest {
+    @Test
+    public void LoginSuccessTest()  {
+
+        openLoginPage();
+        enterEmail("europlatinum@mail.ru");
+        enterPassword("Newjob2!");
+        clickSubmit();
+        String actualText = getSuccessText (".alert.alert-danger");
+        Assert.assertEquals(actualText, "Success");
+    }
+    private String getSuccessText(String s){
```

Comment:
getSuccessText method is wrong. You just return hardcoded test but I guess this method should return a text from UI.